src/js/builtins/shell.ts

```diff
@@ -199,16 +283,19 @@ export function createBunShellTemplateFunction(ShellInterpreter) {
     }
 
     var Shell = function Shell() {
-      const core = new ShellInterpreter(...arguments);
+      const [first, ...rest] = arguments;
+      if (first.raw === undefined) throw new Error("Please use `$` as a tagged template function: $`cmd arg1 arg2`");
```

Comment:
Error message bikeshed. Since we are using the ` to represent the template literal string at the end of the message using it around `$` seems odd. We could just use $ without wrapping it in ` backticks. Also does using "Please" match our error voice (other messages).